src/object/obj_class.c

```diff
@@ -144,13 +168,25 @@ static struct daos_obj_class daos_obj_classes[] = {
 			},
 		},
 	},
+	{
+		.oc_name	= "repl_4_max_rw",
+		.oc_id		= DAOS_OC_R4_MAX_RW,
+		{
+			.ca_schema		= DAOS_OS_STRIPED,
+			.ca_resil		= DAOS_RES_REPL,
+			.ca_grp_nr		= DAOS_OBJ_GRP_MAX,
+			.u.repl			= {
+				.r_num		= 4,
+			},
+		},
+	},
 	{
 		.oc_name	= "repl_max_rw",
 		.oc_id		= DAOS_OC_REPL_MAX_RW,
 		{
 			.ca_schema		= DAOS_OS_STRIPED,
 			.ca_resil		= DAOS_RES_REPL,
-			.ca_grp_nr		= 1,
+			.ca_grp_nr		= DAOS_OBJ_GRP_MAX,
```

Comment:
is there a use case needs both grp_nr and grp_size as MAX?

Reply:
Does it mean we want to have collocated object shards? I'm confused as well

Reply:
sorry im confused as well. no we don't want collocated object shards.
i thought this was to indicate how many targets we could actually use per object on a server.
are you saying this will cause a shard to be replicated on the same target?